lib/p5.play.js

```diff
@@ -4125,7 +4125,13 @@ p5.prototype._warn = function(message) {
    * @param {p5.Transform2D|Sprite} parent
    */
   p5.CollisionShape.prototype.setParentTransform = function(parent) {
-    if (parent instanceof Sprite) {
+    if (parent instanceof Sprite && this instanceof p5.CircleCollider) {
```

Comment:
I'm a little unclear if I should check the above or `parent instanceof Sprite && parent.collider instanceof p5.CircleCollider`